spec/features/course_cloning_spec.rb

```diff
@@ -118,13 +118,14 @@
   end
 
   it 'copies relevant attributes of an existing course with assignments' do
+    pending 'This sometimes fails for unknown reasons.'
```

Comment:
I'd like to avoid this if at all possible. How often does it fail, and what is the reason?

Reply:
So, I've tried running this test 8-10 times and it worked fine so i am pushing it removing pending mark. But my last pr failed because of this only so i am not sure about this. And i am attaching error log too
![Screenshot from 2023-03-08 23-39-38](https://user-images.githubusercontent.com/72390769/223797069-f278dec9-31eb-401b-9132-12f59379514a.png)
So, what happens is that when you select a course to clone from the dropdown, and click on "clone this course" button, it sometimes show some other thing rather than showing "Update Details for cloned Course"

Reply:
This kind of error suggests that the page is in a different state based perhaps on the window size in the test environment, or similar. The 'found 1 time non-visible' suggests that the expected text is getting covered up by something, either because of the window size or because of an open dropdown menu, notification message, or similar.

Reply:
> This kind of error suggests that the page is in a different state based perhaps on the window size in the test environment, or similar. The 'found 1 time non-visible' suggests that the expected text is getting covered up by something, either because of the window size or because of an open dropdown menu, notification message, or similar.

Sometimes it's even 'not visible' like i am updating screenshot below
![Screenshot from 2023-03-09 00-36-14](https://user-images.githubusercontent.com/72390769/223814118-a545f009-7a11-42f2-9331-59b332e62905.png)
Although after my recent commit it didn't threw any error

Reply:
I removed the translation thing from feature spec though because i thought it's useless to check for that text. as we are already checking for checkbox.

Reply:
See @ragesoss it failed again due to that same error. and both of the test cases of this file are failing sometime that's why the other case is pending i guess. Although i did ran the same test around 10times in my local and it was working fine. But it just fails sometimes

Reply:
if you rerun the CI then it might pass too